alfresco-search/src/main/java/org/alfresco/solr/tracker/MetadataTracker.java

```diff
@@ -591,7 +597,20 @@ protected Transactions getSomeTransactions(BoundedDeque<Transaction> txnsFound,
         Transactions transactions;
         // step forward in time until we find something or hit the time bound
         // max id unbounded
-        Long startTime = fromCommitTime == null ? Long.valueOf(0L) : fromCommitTime;
+        Long startTime = fromCommitTime == null  ? 0L : fromCommitTime;
+        log.debug(String.format("#### %s MetadataTracker getSomeTransactions start time: %d end: %d",
+                  this.coreName, 
+                  startTime,
+                  endTime));
+        if(startTime == 0)
```

Comment:
Why in case the start time is 0, we just return those transactions?

Reply:
I think (but I'm not sure) that this is because we don't need to check if we've seen them before if we have `startTime == 0`.

Reply:
When startTime is zero it will loop and call alfresco from epoch (1970) and increment by every hour to fetch all the data. This can be avoided once we are able to get txn details from Alfresco.
By returning a single empty txn we are ending the metadata tracker sooner for it to start again.